examples/experimental/supply_and_demand/market_deep_dive_analysis.ipynb

```diff
@@ -0,0 +1,2213 @@
+    "In this notebook, we perform a deep dive into market dynamics for the Denver metropolitan area (MSA) with a Parcl ID of `5826765`. Additionally, we use the USA as a comparison point in one of the charts, which has already been defined above. However, you can conduct your own analysis by replacing the `parcl_id` with the ID of a market you are interested in exploring. Below, we have code that searches for the 100 largest markets in the country, including the USA.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Retrieve top 100 metro markets, sorted by total population in descending order\n",
+    "metros = client.search.markets.retrieve(\n",
+    "    sort_by='TOTAL_POPULATION',  # Sort by total population\n",
+    "    sort_order='DESC',           # In descending order\n",
+    "    location_type='CBSA',        # Location type set to Core Based Statistical Area (CBSA)\n",
+    "    limit=100                    # Limit results to top 100 metros\n",
+    ")\n",
+    "\n",
+    "# Retrieve national data for the United States to use as a benchmark\n",
+    "us = client.search.markets.retrieve(\n",
+    "    query='United States',  # Query for the United States as a whole\n",
+    "    limit=1                 # Limit results to one (national-level data)\n",
+    ")\n",
+    "\n",
+    "# Concatenate metro market data with national data for comparison\n",
+    "markets = pd.concat([metros, us])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Convert the unique 'parcl_id' values to a list, enabling queries for multiple markets if needed.\n",
+    "market_parcl_ids = markets['parcl_id'].tolist()\n",
+    "\n",
+    "# Check if the market ID designated for analysis is within the list of market IDs.\n",
+    "# Print 'yes' if it is found.\n",
+    "if market_for_analysis_id in market_parcl_ids:\n",
+    "    print('yes')\n",
+    "\n",
+    "# Retrieve the name of the market corresponding to 'market_for_analysis_id'.\n",
+    "# The query filters the 'markets' DataFrame by 'parcl_id' and extracts the market name. We will use this name for labeling purposes.\n",
+    "market_for_analysis_name = markets.query('parcl_id == @market_for_analysis_id')['name'].values[0]\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Retrieve Supply and Demand Data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Define the start date for supply and demand data\n",
+    "start_date = '2022-09-01'\n",
+    "\n",
+    "\n",
+    "# Retrieve the supply (for-sale inventory) data for the market starting from the specified date\n",
+    "supply_df = client.for_sale_market_metrics.for_sale_inventory.retrieve(\n",
+    "    parcl_ids=[market_for_analysis_id,us_parcl_id],\n",
```

Comment:
I would define this once when you first introduce the Denver Parcl ID and US parcl id at the very top. Create the list of `parcl_ids_for_analysis = [market_for_analysis_id, us_parcl_id]` then pass this var to each point instead of redefining it